lib/rspec/rails/configuration.rb

```diff
@@ -105,7 +106,35 @@ def render_views
         end
 
         def render_views?
-          rendering_views
+          rendering_views?
+        end
```

Comment:
`previous definition of use_transactional_fixtures? was here` in [`rspec-core` builds](https://travis-ci.org/github/rspec/rspec-core/jobs/695234603). We'll have to resort to `attr_accessor`.